### PR TITLE
fix(perf): increase MAX_AGE_MINUTES to 120

### PR DESCRIPTION
**Pull request by mxinden**
- When experimenting with increasing iterations, instances are taken down prematurely. See e.g. https://github.com/libp2p/test-plans/actions/runs/5418669631/jobs/9850992882

- When benchmarking from a local developer machine, experimenting for longer is useful.

- The AWS Lambda is only a safety net machanism. In the rare case where we miss to clean up, running the machines for <1h or 2h doesn't matter cost wise.

### CODE REVIEW

File: perf/terraform/modules/ci/cleanup.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ resource "aws_lambda_function" "cleanup" {
     variables = {
       REGIONS         = jsonencode(var.regions)
       TAGS            = jsonencode(var.tags)
-      MAX_AGE_MINUTES = 50
+      MAX_AGE_MINUTES = 120
```
Comment (mxinden) [inline perf/terraform/modules/ci/cleanup.tf:21]: First

File: perf/terraform/modules/ci/files/cleanup.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ Resources:
         Variables:
           REGIONS: '["us-west-2", "us-east-1"]'
           TAGS: '{"Project":"perf", "Name":"node"}'
-          MAX_AGE_MINUTES: '30'
+          MAX_AGE_MINUTES: '120'
```
Comment (mxinden) [inline perf/terraform/modules/ci/files/cleanup.yml:16]: Second

**Review by mxinden — COMMENTED** [review]
@galargh I don't understand why we need two assignments to `MAX_AGE_MINUTES`. Can you comment on this?

File: perf/terraform/modules/ci/files/cleanup.yml (hunk 2 above)
Comment (galargh) [reply perf/terraform/modules/ci/files/cleanup.yml:16]: This YAML is only used for local testing. I just proposed moving the test files to a separate directory to make the two sets more distinct - https://github.com/libp2p/test-plans/pull/216

**Review by galargh — APPROVED** [review]

**Comment by mxinden** [comment]
Merging here. None of these changes affect `multidim-interop/` and thus none of the CI tests apply.

**Comment by mxinden** [comment]
I will apply the changes via terraform. Will report once done.

**Comment by mxinden** [comment]
> I will apply the changes via terraform. Will report once done.

Applied via `perf/config/remote`. State is stored in the S3 bucket.

**Pull request merged**